app/controllers/search_controller.rb

```diff
@@ -5,8 +5,6 @@ class SearchController < ApplicationController
   class InputError < StandardError
   end
 
-  before_action :check_for_login!, only: [:my_plates] # rubocop:todo Rails/LexicallyScopedActionFilter
```

Comment:
Does this remove login requirement for my plates?

Reply:
With that line in it couldn't identify what 'my_plates' was referring to.
There is no reference to 'my_plates' in the code base that I can see, maybe some rails magic that no longer works?